test/integration/good/parser-generator/fun-args-trailing-comma.stan

```diff
@@ -0,0 +1,5 @@
+functions {
+  void foo(real x, real y,){
```

Comment:
Could you also add a test to /bad which shows that something like `void foo(real x, real y ,,)`  [note the multiple , s] fails?